intercom_flutter/android/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 34
```

Comment:
```suggestion
    compileSdk 34
```

Reply:
We can apply this change later as there is also minSdkVersion which will be minSdk